content/en/blog/_posts/2024-xx-xx-matchlabelkeys-podaffinity.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blog
+title: 'Kubernetes 1.31: MatchLabelKeys in PodAffinity graduates to beta'
+date: 2024-XX-XX
+slug: matchlabelkeys-podaffinity
+author: >
+  Kensei Nakada (Tetrate)
+---
+
+Kubernetes 1.29 introduced new fields `MatchLabelKeys` and `MismatchLabelKeys` in PodAffinity and PodAntiAffinity.
+
+In Kubernetes 1.31, this feature moves to beta and the corresponding feature gate (`MatchLabelKeysInPodAffinity`) gets enabled by default.
+
+## `MatchLabelKeys` - accurate scheduling during rolling upgrades
```

Comment:
`accurate` doesn't sound accurate lol... It's more like adding a new feature/knob to adapt to wider/more practical usecases. Can we use a better word here?

Reply:
> ## `MatchLabelKeys` - Enabling version-aware scheduling for rolling upgrades

How does it sound now?

Reply:
How about:

> `MatchLabelKeys` - enhanced scheduling for versatile rolling upgrades

Reply:
Done.